Update README.md (#145)
aredridel/html5: README.md

Interesting features
--------------------

> * Streaming parser: You can use this pass `parser.parse` as an `EventEmitter` and the

Reviewer: ```suggestion
* Streaming parser: You can pass an `EventEmitter` to `parser.parse` and the
```

Author: yaa this would look better
